src/Host.cpp

```diff
@@ -1147,6 +1147,38 @@ bool Host::installPackage( const QString& fileName, int module )
     }
     // reorder permanent and temporary triggers: perm first, temp second
     mTriggerUnit.reorderTriggersAfterPackageImport();
+
+    // This event was inspired by "Zoilder" in a Mudlet derivative in 2015
```

Comment:
Do you still want me to take out these comments, I'd prefer to leave them, as I said, as the record of where they came from. (It would be likely that there are more to come as we try and reverse engineer what Zoilder has done - especially as I think there are one or two imperfections or limitations in their work!)